db/seeds.rb

```diff
@@ -1,10 +1,29 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+# This file adds seeds by parsing the JSON files in the db/seeds/ folder.
+
+# If a "date_flag" is set for a seed, we generate the dates dynamically.
+# This ensures a seed which is always intended to be in a time-sensitive state
+# remains in that state without having to manually update the dates.
+
+def registered_date(flag)
+  dates = {
+    expired: 37.months.ago,
```

Comment:
Should these not be making use of the config values for expiry and renewal window to work out how many months to use? Or have I missed something?

Reply:
@Cruikshanks They should and that makes complete sense. I'll tweak that now.

Reply:
@Cruikshanks Should be sorted now. It's a little awkward to read but hopefully the comments clear it up.